assets/dev/scss/admin/admin.scss

```diff
@@ -10,6 +10,9 @@
 
 // Admin Global
 @import "global";
+@import "../global/dialog-base";
```

Comment:
I have preloaded the dialog for the admin and editor part, because the Asset Loader only works for the frontend.

Is this okay?